src/components/TimeSeriesCard/TimeSeriesCard.story.jsx

```diff
@@ -993,7 +993,7 @@ storiesOf('Watson IoT|TimeSeriesCard', module)
                 details: 'Alert name',
               },
               {
-                startTimestamp: 1572804320000,
+                startTimestamp: 1572313622000,
```

Comment:
Despite this, I'm not seeing any alerts on the `highlight alert ranges` story?

![image](https://user-images.githubusercontent.com/3360588/74377329-53440580-4da9-11ea-8600-2b7b5d1f5682.png)

Reply:
there's a weird bug with the time series stories where sometimes the chart is rendering a previous chart.  Try and force refresh your page and you should see the Alert Ranges I think.

Reply:
@scottdickerson Got it! Could you enter an issue for that timeseries chart rendering bug?